Error on Display Owner

Caused by: org.thymeleaf.exceptions.TemplateProcessingException: Exception evaluating SpringEL expression: "pet.type" (template: "owners/ownerDetails" - line 52, col 21)
	at org.thymeleaf.spring5.expression.SPELVariableExpressionEvaluator.evaluate(SPELVariableExpressionEvaluator.java:292) ~[thymeleaf-spring5-3.0.12.RELEASE.jar:3.0.12.RELEASE]
	at org.thymeleaf.standard.expression.VariableExpression.executeVariableExpression(VariableExpression.java:166) ~[thymeleaf-3.0.12.RELEASE.jar:3.0.12.RELEASE]
	at org.thymeleaf.standard.expression.SimpleExpression.executeSimple(SimpleExpression.java:66) ~[thymeleaf-3.0.12.RELEASE.jar:3.0.12.RELEASE]
	at org.thymeleaf.standard.expression.Expression.execute(Expression.java:109) ~[thymeleaf-3.0.12.RELEASE.jar:3.0.12.RELEASE]
	at org.thymeleaf.standard.expression.Expression.execute(Expression.java:138) ~[thymeleaf-3.0.12.RELEASE.jar:3.0.12.RELEASE]
	at org.thymeleaf.standard.processor.AbstractStandardExpressionAttributeTagProcessor.doProcess(AbstractStandardExpressionAttributeTagProcessor.java:144) ~[thymeleaf-3.0.12.RELEASE.jar:3.0.12.RELEASE]
	at org.thymeleaf.processor.element.AbstractAttributeTagProcessor.doProcess(AbstractAttributeTagProcessor.java:74) ~[thymeleaf-3.0.12.RELEASE.jar:3.0.12.RELEASE]

--- a/pet-clinic-data/src/main/java/guru/springframework/sfgpetclinic/model/PetType.java
+++ b/pet-clinic-data/src/main/java/guru/springframework/sfgpetclinic/model/PetType.java
@@ -14,7 +14,6 @@ import javax.persistence.Table;
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
-@ToString
 @Entity
 @Table(name = "types")
 public class PetType extends BaseEntity {
@@ -22,4 +21,8 @@ public class PetType extends BaseEntity {
     @Column(name = "name")
     private String name;
 
+    @Override
+    public String toString() {
+        return name;
+    }
 }

--- a/pet-clinic-web/src/main/resources/templates/owners/ownerDetails.html
+++ b/pet-clinic-web/src/main/resources/templates/owners/ownerDetails.html
@@ -49,7 +49,7 @@
                 <dd
                         th:text="${#temporals.format(pet.birthDate, 'yyyy-MM-dd')}" /></dd>
                 <dt>Type</dt>
-                <dd th:text="${pet.type}" /></dd>
+                <dd th:text="${pet.petType}" /></dd>
             </dl>
         </td>
         <td valign="top">

